docs/nGQL-tutorial.md

```diff
@@ -98,6 +98,15 @@ SHOW SPACES
 ----------------
 ```
 
+```
+SHOW CREATE SPACES myspace_test
+------------------------------------------------------------------------
+| Space        | Create Space                                           |
+------------------------------------------------------------------------
+| myspace_test | CREATE SPACE s (partition_num = x, replica_factor = x) |
```

Comment:
CREATE SPACE `myspace_test` (partition_num = x, replica_factor = y)

BTW, this looks a bit different from mysql manual.

Reply:
Yeah, I update it  following the output of nebula

Reply:
Good catch! 👍